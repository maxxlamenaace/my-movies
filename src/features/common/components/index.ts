import Loader from './loader/Loader';
import Logo from './logo/Logo';
import Footer from './footer/Footer';
import AppBar from './app-bar/AppBar';
import Container from './container/Container';
import ScrollableAppBar from './scrollable-app-bar/ScrollableAppBar';
import Sidebar from './sidebar/Sidebar';
import ManualSwiper from './manual-swiper/ManualSwiper';
import Wrapper from './wrapper/Wrapper';
import PaginatorSwiper from './paginator-swiper/PaginatorSwiper';

export {
  Loader,
  Logo,
  Footer,
  AppBar,
  Container,
  ScrollableAppBar,
  Sidebar,
  ManualSwiper,
  Wrapper,
  PaginatorSwiper,
};
