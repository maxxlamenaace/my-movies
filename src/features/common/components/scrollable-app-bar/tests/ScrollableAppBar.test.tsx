import React from 'react';
import { cleanup, render } from '@testing-library/react';

import { ScrollableAppBar } from '@/features/common';

describe('ScrollableAppBar', () => {
  afterEach(() => {
    cleanup();
  });

  it('renders correctly', () => {
    const { asFragment } = render(
      <ScrollableAppBar>
        <div id='children' />
      </ScrollableAppBar>,
    );
    expect(asFragment()).toMatchSnapshot();
  });

  it('renders with children', () => {
    const { container } = render(
      <ScrollableAppBar>
        <div id='children' />
      </ScrollableAppBar>,
    );
    expect(container.querySelector('#children')).toBeDefined();
  });
});
