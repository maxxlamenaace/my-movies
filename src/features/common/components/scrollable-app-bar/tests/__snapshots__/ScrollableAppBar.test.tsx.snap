// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollableAppBar renders correctly 1`] = `
<DocumentFragment>
  <div
    id="children"
    sx="[object Object]"
  />
</DocumentFragment>
`;
